ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -273,8 +471,8 @@ KEXT_STATIC int HandleVnodeOperation(
                 KAUTH_VNODE_SEARCH |
                 KAUTH_VNODE_READ_SECURITY |
                 KAUTH_VNODE_READ_ATTRIBUTES |
-                KAUTH_VNODE_READ_EXTATTRIBUTES |
-                KAUTH_VNODE_DELETE))
+                KAUTH_VNODE_READ_EXTATTRIBUTES) ||
+            (isDeleteAction && isRename))
```

Comment:
Ah indeed, good idea. I think I defaulted it to `true` originally because the version check was somewhere else than it now is, but with the current code, a default of `false` does indeed make it simpler.